lib/galaxy/datatypes/binary.py

```diff
@@ -2730,12 +2697,12 @@ def __init__(self, **kwd):
         super().__init__(**kwd)
         self._magic = binascii.unhexlify("6be33e6d47530e3c")
 
-    def sniff(self, filename):
+    def sniff_prefix(self, sniff_prefix):
         # The first 8 bytes of any daa file are 0x3c0e53476d3ee36b
```

Comment:
Not related to the PR but why is this the same as unhexlify input, but reversed?

Reply:
I don't know - it does seem odd.

Reply:
little/big endian probably.

Reply:
blame Gulliver